### PR TITLE
Hook for generating values for conversion between crm.lead and res.partner

### PR DESCRIPTION
**Pull request by yajo**
Relaunch https://github.com/odoo/odoo/pull/8899 to master.

I think I broke that PR somehow when rebasing.

### CODE REVIEW

File: addons/crm/crm_lead.py [hunk 1]
```diff
@@ -711,28 +711,35 @@ def convert_opportunity(self, cr, uid, ids, partner_id, user_ids=False, team_id=
...
-            'street': lead.street,
-            'street2': lead.street2,
-            'zip': lead.zip,
-            'city': lead.city,
-            'country_id': lead.country_id and lead.country_id.id or False,
-            'state_id': lead.state_id and lead.state_id.id or False,
-            'is_company': is_company,
-            'type': 'contact'
-        }
+        vals = lead._map_values_to_partner(name, is_company, parent_id)[0]
```
Comment (pedrobaeza) [inline addons/crm/crm_lead.py]: Why the final [0]?

Comment (pedrobaeza) [inline addons/crm/crm_lead.py]: This raises an error

**Comment by Yajo** [comment]
This is ready now.

**Comment by pedrobaeza** [comment]
:+1:

**Comment by wtaferner** [comment]
Maybe you could put everything into one commit, but anyway it looks good to me.

**Comment by carlos-incaser** [comment]
:+1:

**Comment by rafaelbn** [comment]
:+1:

**Comment by tivisse** [comment]
Hi @Yajo 

It seems that it has been done during that time in the master version ! :)

See: https://github.com/odoo/odoo/blob/master/addons/crm/models/crm_lead.py#L764

So I close this PR.

Thanks for the feedback anyway,

Yannick.

**Pull request closed without merging**